### PR TITLE
Parse load offset on project loading

### LINKED ISSUE #11385: Can`t restore offset when loading project
Windows Cutter 1.7.1 64b

Project loads with zero data offset.
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Gentoo PPC 32  Big
| File format of the file you reverse (mandatory)      |  raw kernel image(memory map) from uImage
| Architecture/bits of the file (mandatory)            | PPC/32
| r2 -v full output, **not truncated** (mandatory)     |  r2-2.8.0-189-gf82b28982
### Expected behavior
When loading project it must restore loaded file offset 

### Actual behavior
Data loaded at ZERO offset

### Steps to reproduce the behavior 
start r2, load image with offset, save project, load saved project - now we have loaded image without offset.

### PR DESCRIPTION
**Pull request by andryblack**
fix #11385

### CODE REVIEW

File: libr/core/project.c [hunk 1]
```diff
@@ -366,6 +366,32 @@ R_API RThread *r_core_project_load_bg(RCore *core, const char *prjName, const ch
 
 /*** ^^^ thready ***/
 
+
+static ut64 getProjectLaddr(RCore *core, const char *prjfile) {
+	ut64 laddr = 0;
+	FILE *fd;
+	char buf[256];
+	fd = r_sandbox_fopen (prjfile, "r");
```
Comment (radare) [inline libr/core/project.c]: define and assign in the same line

Comment (radare) [reply libr/core/project.c]: what about r_file_slurp + strstr ("\n\"e bin.laddr =")? will be faster and shorter imho

File: libr/core/project.c [hunk 2]
```diff
@@ -438,25 +464,23 @@ R_API bool r_core_project_open(RCore *core, const char *prjfile, bool thready) {
...
-			if (!r_core_file_open (core, filepath, 0, 0)) {
+			if (!r_core_file_open (core, filepath, 0, UT64_MAX)) {
 				eprintf ("Cannot open file '%s'\n", filepath);
 				ret = false;
 				goto beach;
 			}
 		}
 	}
-
+	mapaddr = getProjectLaddr(core, prj);
```
Comment (radare) [inline libr/core/project.c]: missing space before (

File: libr/core/project.c [hunk 3]
```diff
@@ -438,25 +464,23 @@ R_API bool r_core_project_open(RCore *core, const char *prjfile, bool thready) {
...
+	if (mapaddr) {
+		r_config_set_i (core->config, "bin.laddr", mapaddr);
+	}
 	if (filepath[0] && close_current_session && r_config_get_i (core->config, "file.info")) {
 		mapaddr = r_config_get_i (core->config, "file.offset");
 		(void)r_core_bin_load (core, filepath, mapaddr? mapaddr: UT64_MAX);
 	}
-	if (thready) {
```
Comment (radare) [inline libr/core/project.c]: why kill the thready project loading support?

Comment (andryblack) [reply libr/core/project.c]: already killed at line 404

File: libr/core/project.c (hunk 1 above)
Comment (andryblack) [reply libr/core/project.c]: copied from r_core_project_info, for good need preload some vars with single pass

**Pull request merged**